src/uk/gov/hmcts/contino/AppPipelineDsl.groovy

```diff
@@ -40,6 +40,11 @@ class AppPipelineDsl extends CommonPipelineDsl implements Serializable {
     config.crossBrowserTest = true
   }
 
+  void enableCrossBrowserTest(List<String> browsers, int timeout = 120) {
```

Comment:
Do u still need this method ? https://github.com/hmcts/cnp-jenkins-library/pull/714/files#diff-08ec10cb390079edcd7ae93cb8d29b85e267a1ed73f3e16c3248af11ce5855efR38